### DISCUSSION TITLE
SQL Injection advanced lesson 5 hints

### DISCUSSION THREAD

**Discussion by olga-in-clouds** [WebGoat/WebGoat#1726] [Q&A]
Hello, 

For the advanced SQL Injection exercise 5 the following hints are provided:

![image](https://github.com/WebGoat/WebGoat/assets/8805399/fcde8e8f-93a2-42c7-98f0-e3df4ebd7287)

![image](https://github.com/WebGoat/WebGoat/assets/8805399/028a5b2f-c3c2-424d-8c1c-cacc8ac9e2c1)

![image](https://github.com/WebGoat/WebGoat/assets/8805399/214a1482-b955-487b-9dd6-ddf4d665929f)

![image](https://github.com/WebGoat/WebGoat/assets/8805399/9bbe06e2-607c-4d81-a5be-55c4c6e01519)

However, the solution provided in the [solution section](https://github.com/WebGoat/WebGoat/wiki/Main-Exploits) is different:

![image](https://github.com/WebGoat/WebGoat/assets/8805399/b7e08005-8f73-4f16-8aa0-35d984ed1898)

Are the hints valid?

**Comment by nbaars** [comment]
@olga-in-clouds fixed with #2045
